config/initializers/filter_parameter_logging.rb

```diff
@@ -11,6 +11,29 @@
   to_date
   qqtotalfilesize
   type
+  folder_id
+  startDate
+  endDate
+  included
+  page
+  useCache
+  number
+  size
+  sort
+  showCompleted
+  excludeProvidedMessage
+  document_id
+  body
+  category
+  subject
+  cookie_id
+  reply_id
+  ids
+  code
+  endpoint_sid
+  message_id
+  os_name
+  filter
```

Comment:
@cadibemma Would any of these have PII included?

Reply:
The only one in my opinion with potential to have PII included is 'body' (So I just removed it).